src/main/java/MockClass.java

```diff
@@ -1,8 +1,25 @@
 
 public class MockClass {
 
+    /**
+     * cioc
+     * @param seed
+     * @return
+     */
     public int doSomething(int seed) {
 
         return seed + 5;
     }
+
+    public static void main(String args[]) {
+        try {
+            doNothing();
+        } catch (Throwable t) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Catch Exception instead of Throwable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.tiolktrack.byznass.com:9000/coding_rules#rule_key=squid%3AS1181)